activerecord/lib/active_record/associations.rb

```diff
@@ -71,6 +71,20 @@ def initialize(reflection = nil)
     end
   end
 
+  class InverseOfAssociationNotBidirectionalError < ActiveRecordError # :nodoc:
```

Comment:
I think there was already some discussion about the name, but I prefer `InverseOfAssociationNotFoundError`, as it's explicit that the "inverse of" doesn't exist.